File: Medium/223. Rectangle Area.cpp
class Solution {
public:
    int computeArea(int A, int B, int C, int D, int E, int F, int G, int H) {
        long allArea = (long)(C-A)*(D-B) + (G-E)*(H-F);
		long overW = max(0L, min(C,G) - (long)max(A,E));
        long overH = max(0L, min(D,H) - (long)max(B,F));
		return allArea - overW*overH;
    }
};

File: Medium/223. Rectangle Area.py
class Solution:
    def computeArea(self, A: int, B: int, C: int, D: int, E: int, F: int, G: int, H: int) -> int:
        allArea = (C-A)*(D-B) + (G-E)*(H-F)
        overW = max(0, min(C,G) - max(A,E))
        overH = max(0, min(D,H) - max(B,F))
        return allArea - overW * overH